python/hail/ldMatrix.py

```diff
@@ -83,3 +84,110 @@ def read(path):
 
         jldm = Env.hail().methods.LDMatrix.read(Env.hc()._jhc, path)
         return LDMatrix(jldm)
+
+    @typecheck_method(path=strlike,
+                      column_delimiter=strlike,
+                      header=nullable(strlike),
+                      parallel_write=bool,
+                      entries=enumeration('full', 'lower', 'strict_lower', 'upper', 'strict_upper'))
+    def export(self, path, column_delimiter, header=None, parallel_write=False, entries='full'):
+        """Exports this matrix as a delimited text file.
+
+        **Examples**
+
+        Write a full LD matrix as a tab-separated file:
+
+        >>> vds.ld_matrix().export('output/ld_matrix.tsv', column_delimiter='\t')
+
+        Write a full LD matrix as a comma-separated file with the variant list as a header:
+
+        >>> ldm = vds.ld_matrix()
+        >>> ldm.export('output/ld_matrix.tsv',
+        ...            column_delimiter=',',
+        ...            header=','.join(ldm.variant_list))
```

Comment:
You're missing the parentheses after `variant_list`. That should fix your tests.